db/migrate/20220923062245_create_table_deals.rb

```diff
@@ -0,0 +1,19 @@
+class CreateTableDeals < ActiveRecord::Migration[7.0]
+  def change
+    create_table :table_deals do |t|
+      t.string :title
+      t.string :description
+      t.integer :price_in_cents
+      t.integer :discount_price_in_cents
+      t.integer :quantity
+      t.boolean :is_publishable
+      t.decimal :tax_percentage
+      t.integer :created_by 
```

Comment:
t.references ?

Reply:
If I had used t.references then it would create a column with a name as `created_by_id`